crates/starknet-os/src/config.rs

```diff
@@ -67,6 +69,24 @@ const fn default_use_kzg_da() -> bool {
     true
 }
 
+/// un-gzips the default OS that is baked into the binary.
+///
+/// Uncompressed, this file is roughly 40M, and compresses effectively down to less than 1M.
+/// So it is included in the binary in gzipped form with this fn provided to reinflate it.
+///
+/// NOTE: the results are not cached, so this will always be an expensive call.
+pub fn gunzip_default_os() -> std::io::Result<String> {
```

Comment:
Shouldn't this return `Vec<u8>` to mirror the old `DEFAULT_COMPILED_OS` and the fact `run_os` types in bytes?

Reply:
Good idea, added in [2c4bf97](https://github.com/keep-starknet-strange/snos/pull/369/commits/2c4bf972a49748b084e4bc99b22c7dd39fc94746)